examples/WktExporter.js

```diff
@@ -0,0 +1,120 @@
+/*
```

Comment:
The stars are not aligned in this header. The copyright year should be 2015-2018 now.